apps/form-system/web/src/lib/routes.tsx

```diff
@@ -0,0 +1,37 @@
+import { Outlet, RouteObject } from 'react-router-dom'
+import { HeaderInfoProvider } from '../context/HeaderInfoProvider'
+import { Application } from '../routes/Application'
+import { Applications } from '../routes/Applications'
+import { Layout } from '../components/Layout/Layout'
+
+export const BASE_PATH = '/form'
+
+export const routes: RouteObject[] = [
+  {
+    element: (
+      <HeaderInfoProvider>
+        <Layout>
+          <Outlet />
+        </Layout>
+      </HeaderInfoProvider>
+    ),
+    children: [
+      {
+        children: [
+          {
+            path: '/:slug',
+            element: <Applications />
+          },
+          {
+            path: '/:slug/:id',
+            element: <Application />
+          }
+        ]
+      },
+      {
+        path: '*',
+        element: <>Dummy</>
```

Comment:
You should add a Not found route here. I created a reusable Problem component sometime ago for all kinds of problems. You could use the NotFound component from here
`libs/portals/core/src/screens/NotFound.tsx`.

Also is there a plan on protecting the form scope wise. Because if so you should look into displaying the AccessDenied component when user does not have rights to work on an application
`libs/portals/core/src/screens/AccessDenied.tsx`